tree-sitter/src/TreeSitter/Unmarshal.hs

```diff
@@ -221,7 +221,8 @@ getFields = go Map.empty -- >>= \fields -> liftIO (print (Map.keys fields)) >> p
               keepGoing <- step
               let fs' = case fieldName of
                     Just fieldName' -> Map.insertWith (flip (++)) fieldName' [node'] fs
-                    _ -> if nodeIsNamed node' /= 0
+                    -- NB: We currently skip “extra” nodes (i.e. ones occurring in the @extras@ rule), pending a fix to https://github.com/tree-sitter/haskell-tree-sitter/issues/99
+                    _ -> if nodeIsNamed node' /= 0 && nodeIsExtra node' == 0
                       then Map.insertWith (flip (++)) (FieldName "extraChildren") [node'] fs
                       else fs
```

Comment:
The addition of the `nodeIsExtra` test means that we just silently drop such fields. It won’t be a big change to nest the conditionals and add extras into a dedicated `extras` field instead, once we’re fixing #99. @maxbrunsfeld made the (v. good) point that having both `extras` and `extraChildren` fields is gonna be super confusing, so maybe we should also rename the latter to `otherChildren` at that point.